docs/metrics.md

```diff
@@ -98,11 +112,11 @@ For our purposes, we will use the `extra` field for a few purposes:
 
 -   To log the FxA user id of the client logging the event (e.g. `"fxa_uid": uid`)
 -   To log the UUID of the item that has been added or changed (e.g. `"item_id": UUID`)
--   To log the fields that are modified when an item is updated in the datastore (e.g. `"fields":"password,notes"`  (because the value has to be a string we will have to concat the fields that were updated somehow)
+-   To log the fields that are modified when an item is updated in the datastore (e.g. `"fields": "password,notes"`  (because the value has to be a string we will have to concat the fields that were updated somehow)
```

Comment:
Looking at this again, I note the backed has some quirks with what the user sees versus what's recorded.  The JSON structure (before encryption) looks something like (`...` added for data that's likely not relevant):

```javascript
{
  "id": <uuid>,
  "title": <user string>,
  "origins": [0..n<URL string>],
  "tags": [0..n<user token>],
  ...,
  "entry": {
    "kind": "login",
    "username": <user string>,
    "password": <user string>,
    "notes": <user string>
  }
  "
}
```

It would be convenient if we could allow for these quirks in the metrics model.  For example, instead of:

```
"fields": "title,origin,password,notes"
```

if we could instead emit:

```
"fields": "title,origins,login.pasword,login.notes"
```